CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 #### Data migrations
 - [#1009](https://github.com/LayerManager/layman/issues/1009) Fill column `description` in `publications` table in prime DB schema. Value is taken from GeoServer for layers and from filesystem for maps.
 ### Changes
+- [#1048](https://github.com/LayerManager/layman/issues/1048) New key `wfs`.`name` was added to [GET Workspace Layer](doc/rest.md#get-workspace-layer) response.
+- [#1048](https://github.com/LayerManager/layman/issues/1048) Names of [Feature types and Layers in GeoServer WFS workspace](doc/data-storage.md#geoserver) are `l_<uuid`.
```

Comment:
```suggestion
- [#1048](https://github.com/LayerManager/layman/issues/1048) Names of [Feature types and Layers in GeoServer WFS workspace](doc/data-storage.md#geoserver) are `l_<uuid>`.
```

Reply:
Anyway, I prefer `l-<uuid>` (dash instead of underscore), to be consistent with UUID part separator.